distribution/docs/src/main/resources/_contents/_configuring/configuring-from-import-contents.adoc

```diff
@@ -0,0 +1,78 @@
+This is useful for migrating from one running instance to another.
+
+To export the current system configurations, follow these instructions:
+
+. Select the `System` tab (next to the Applications tab) image:exporting_configuration_step1.png[Exporting Step 1,${image.width}]
+. Click the `Export Configuration` button image:exporting_configuration_step2.png[Exporting Step 2,${image.width}]
+. Fill out the form, specifying the _destination_ for the export.  A relative path will be relative to `<${branding}_HOME>`. image:exporting_configuration_step3.png[Exporting Step 3,${image.width}]
+. Click the `Start Export` button
+. If there are no warnings or errors, the form will automatically close upon finishing the export
+
+==== Export Existing Configuration Settings from ${command-console}
+
+To export the current ${branding} configuration:
+
+. Using the ${command-console}, type in `migration:export <directory>`. This command creates the exported configuration files that are saved to the specified directory. If no directory is specified it will default to `<${branding}_HOME>/etc/exported`
+. Zip up the exported files in the export directory.
+----
+cd  <${branding}_HOME>/etc/exported
+zip exportedFiles.zip *
+----
+
+==== Troubleshooting Common Warnings or Failures
+
+If export is unsuccessful, use this list to verify the correct configuration.
+
+===== Insufficient Write Permissions
+
+In the following case, the directory the user tried to export to had permissions set to read only.
+image:exporting_configuration_error.png[Exporting Error,${image.width}]
+
+===== Properties Set to Absolute File Paths
+
+In the following case, the user had a property set to an absolute file path.
+This is not allowed, and can be fixed by updating the property to a value that is relative to ${branding} home.
+However, notice that the export did not completely fail.
+It is simply informing the user that they did not include a specific file.
+image:exporting_configuration_warning.png[Exporting Warning,${image.width}]
+
+[IMPORTANT]
+====
+Some system configuration files contain paths to other configuration files. For instance, the `system.properties` file contains the `javax.net.ssl.keyStore` property which provides the path to system key store.
+The files referred to in the system configuration files will be included in the export process only if the path is relative to `${branding}_HOME`.
+Using absolute paths and/or symbolic links in those cases will cause the `migration:export` command to display warnings about those files and exclude them from the export process.
+The export process itself will not be aborted.
+====
+
+==== Importing the Exported Configuration Settings
+
+[WARNING]
+====
+Importing of configurations must take place before starting the new installation for the first time.
+====
+
+To import a previously exported configuration:
+
+. Delete all exising `.config` files from `<${branding}_HOME>/etc`: `rm <${branding}_HOME>/etc/*.config`
```

Comment:
The actual list of commands should be:

```
cd <{$branding}_HOME>
rm etc/*.config
unzip /path/to/exported/zip/exportFiles.zip
```

We should also add a note saying that the command will confirm file overwrite and that all files should be overwritten.